.github/workflows/tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           path: deps
           key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
       - name: <elixir> Install dependencies (if needed)
-        if: steps.elixir-cache.outputs.cache-hit != 'true'
+        if: steps.asdf-cache.outputs.cache-hit != 'true'
```

Comment:
This isn't quite right: it should be split into two steps. The first one installs Hex/Rebar (and uses the ASDF cache), and the second installs the dependencies (and uses the Elixir cache).

Reply:
I tried to fix this according to your comment here. Can you let me know if that's what you had in mind?

Reply:
@losvedir yep, that's perfect